### ISSUE TITLE
Undocumented method of electron's webFrame object used.

### ISSUE THREAD

**Issue by revant117** [wavebox/waveboxapp#753]
* Wavebox Version: [3.14.6]
* Operating System: [Windows 10]

I have gotten stuck at this part of wavebox's source. 

https://github.com/wavebox/waveboxapp/blob/master/src/guest/src/Browser/Extensions/CRExtensionLoader.js#L109

I have tried to look for a way to figure out what is exactly going on but could not find any reference to `createContextId` method. 
Any sort of guidence from the wavebox team would be apriciated.

**Comment by Thomas101** [comment]
The method atomically creates a new JS context id for the isolate to run the extension content-script. Are you having any trouble with it?

**Comment by revant117** [comment]
Thanks, @Thomas101 for such a quick reply. Not having any problems as such. I was trying to understand what was going on there. I got that it is creating a context id, but what confused me is that I could not find a reference `webFrame.createContextId` in electron's source or documentation. Is webFrame object being overridden? I was just trying to understand the code base so that I can modify and eventually contribute. But I got a little stuck at this point.
